test/core/storage/car/genesis_test.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+
+#include "primitives/block/block.hpp"
+#include "storage/car/car.hpp"
+#include "storage/ipfs/impl/in_memory_datastore.hpp"
+#include "testutil/outcome.hpp"
+#include "testutil/read_file.hpp"
+#include "testutil/resources/resources.hpp"
+#include "vm/actor/builtin/account/account_actor.hpp"
+#include "vm/actor/builtin/init/init_actor.hpp"
+#include "vm/actor/builtin/market/actor.hpp"
+#include "vm/actor/builtin/miner/types.hpp"
+#include "vm/actor/builtin/reward/reward_actor.hpp"
+#include "vm/actor/builtin/storage_power/storage_power_actor_state.hpp"
+
+/**
+ * @given genesis file
+ * @when decode
+ * @then success
+ */
+TEST(GenesisTest, Decode) {
```

Comment:
The test contains 5 todos (!), please create a task in jira for fixing the test and leave its id alongside each todo.

Reply:
Please consider disabling the whole test while todos are not implemented.